source/crud/tests/unified/estimatedDocumentCount.yml

```diff
@@ -144,16 +122,11 @@ tests:
         events:
           - commandStartedEvent:
               command:
-                aggregate: *collection0Name
-                pipeline: &pipeline
-                  - $collStats: { count: {} }
-                  - $group: { _id: 1, n: { $sum: $count }}
-              commandName: aggregate
+                count: *collection0Name
+              commandName: count
               databaseName: *database0Name
 
-  - description: "estimatedDocumentCount uses count on less than 4.9.0"
-    runOnRequirements:
-      - maxServerVersion: "4.8.99"
+  - description: "estimatedDocumentCount uses count"
```

Comment:
this and the tests below seem like they are redundant with the tests above, now that the behavior is unified below and above 4.9?

Reply:
Good catch, fixed.